src/native.js

```diff
@@ -163,10 +163,41 @@ export function getNativeTargeting(bid) {
       value = value.url;
     }
 
+    const sendPlaceholder = deepAccess(
+      bidReq,
+      `mediaTypes.native.${asset}.usePlaceholder`
```

Comment:
Any new thoughts on the name for this property?  I know we were throwing other ideas around in a different discussion.

cc @mkendall07